tests/fixtures/scenarios/vendor/vendor.yaml

```diff
@@ -39,6 +39,14 @@ spec:
         - demo
       targets:
         - "components/terraform/{{ .Component }}/{{.Version}}"
+        
+    - component: "mock"
```

Comment:
There’s no harm vendor during this weather component. What we want to avoid is any terraform plan or apply on the weather component. Venting the component on the other hand isn’t a great test of veneering because there’s almost no files to vendor.